pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -39,6 +39,7 @@ const (
 	InfraServiceIngressIPControllerServiceAccountName           = "service-ingress-ip-controller"
 	InfraPersistentVolumeRecyclerControllerServiceAccountName   = "pv-recycler-controller"
 	InfraResourceQuotaControllerServiceAccountName              = "resourcequota-controller"
+	InfraHorizontalPodAutoscalerControllerServiceAccountName    = "horizontal-pod-autoscaler"
```

Comment:
move to a separate commented section and indicate that is a special upstream controller that we're adding additional permissions to.